plugins/community/infobloxopen-gorm/v1.1.2/buf.plugin.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/community/infobloxopen-gorm
+plugin_version: v1.1.2
+source_url: https://github.com/infobloxopen/protoc-gen-gorm
+description: Generate GORM models and APIs for simple object persistence tasks.
+spdx_license_id: Apache-2.0
+license_url: https://github.com/infobloxopen/protoc-gen-gorm/blob/v1.1.2/LICENSE
+output_languages:
+  - go
```

Comment:
Can you also add this:

```yaml
deps:
  - plugin: buf.build/protocolbuffers/go:v1.28.1
```

It looks like the generated code depends on the Go generated base types, so this will help provide a more complete buf.gen.yaml example in the UI (by listing dependencies).